test/integration/index.test.js

```diff
@@ -20,11 +20,16 @@ describe('run-elm', () => {
 
       let result;
       try {
-        result = await execFile(runElmPath, input, { cwd: projectDir });
+        result = await execFile(runElmPath, input, {
+          cwd: projectDir,
+          maxBuffer: 1024 * 1024 * 100
+        });
       } catch (e) {
-        const { code, stderr } = e;
-        throw new Error(`process timeout or non-zero exit code ${code}${stderr ? `: ${stderr}` : ''}`);
+        const { code, stderr, message } = e;
+        const details = stderr || message;
```

Comment:
When `stdout` exceeds `maxBuffer`, we get error details in `e.message` instead of `e.stdout`. This situation does not happen here ATM because `maxBuffer` is always larger than we need, but it's still useful to have things written correctly for future cases.